generators/app/tasks/installDependencies.js

```diff
@@ -96,6 +96,7 @@ module.exports = function installDependencies() {
   if (this.features.googleanalytics) {
     DEPENDENCIES.push('react-native-google-analytics-bridge');
     DEPENDENCIES.push('redux-beacon');
+    DEPENDENCIES.push('@redux-beacon/react-native-google-analytics');
```

Comment:
Dunno. Shouldn't these both go with `@` ?

Reply:
Nope.. Yeah, it's weird, but that's in the documentation